src/emoji/emojiSelectors.js

```diff
@@ -27,3 +27,11 @@ export const getActiveRealmEmojiTransformed = createSelector(getActiveRealmEmoji
   });
   return transformedEmojis;
 });
+
+export const getAllRealmEmojiTransformed = createSelector(getAllRealmEmoji, emojis => {
```

Comment:
I don't think the name is telling me enough. 'transformed' is too generic. Can you come up with a better name?